### PR TITLE
Fix single quotes dropped

### PR DESCRIPTION
**Pull request by Greendor1234**
To close issue #151

According to what it's written in the comments, I introduced a regular expression that replaces the quotes for anything with {{}}.

I checked the results, it seems to work.

### CODE REVIEW

File: pkg/processor/deployment/deployment.go [hunk 1]
```diff
@@ -129,7 +130,8 @@ func (d deployment) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstr
 		return true, nil, err
 	}
 
-	spec = strings.ReplaceAll(spec, "'", "")
+	r := regexp.MustCompile(`'({{((.*|.*\n.*))}}.*)'`)
+	spec = r.ReplaceAllString(spec, "${1}")
```
Comment (arttor) [inline pkg/processor/deployment/deployment.go:134]: it looks like exacltly what we need but for me it is always hard read regexp. Can you please move this `'` replacement code to a separate function and add table unit test covering edge cases?

**Review by arttor — CHANGES_REQUESTED** [review]
Overall looks really good, thank you for adjusting test examples. I had only one comment related to unit tests.

**Comment by Greendor1234** [comment]
I added all the edge cases that came to my mind

**Review by arttor — APPROVED** [review]

**Comment by arttor** [comment]
@Greendor1234 thank you for your contribution. The changes are available in [v0.4.15 release](https://github.com/arttor/helmify/releases/tag/v0.4.15).
